tpl/cast/cast.go

```diff
@@ -30,6 +30,22 @@ type Namespace struct {
 
 // ToInt converts the given value to an int.
 func (ns *Namespace) ToInt(v interface{}) (int, error) {
+	v = convertTemplateToString(v)
```

Comment:
Please don't change exisiting methods. This doesn't look correct and I'm surprised there were not red tests.

Reply:
> Please don't change exisiting methods

Why not? Should I just copy-and-paste the code that was used in toInt? I just took what was there before and moved it into a separate helper function.